packages/runtime-core/__tests__/errorHandling.spec.ts

```diff
@@ -431,5 +432,32 @@ describe('error handling', () => {
     log.mockRestore()
   })
 
+  it('app-level error handling', () => {
```

Comment:
duplicate of
https://github.com/vuejs/vue-next/blob/master/packages/runtime-core/__tests__/apiApp.spec.ts#L299

Reply:
In `errorHandling.ts`, errors which is not captured will be handled by appErrorHandler, I want to test the logic of this branch, so it is not the same.

Reply:
Hmm I don't see how this is different from the test in `apiApp.spec.ts`. The current coverage looks fine: https://vue-next-coverage.netlify.com/runtime-core/src/errorhandling.ts